apps/gallery/js/ImageEditor.js

```diff
@@ -226,7 +226,7 @@ var exposureSlider = (function() {
     if (!dragStart)
       dragStart = currentExposure;
 
-    setExposure(dragStart + exposureDelta);
+    setTimeout(function() { setExposure(dragStart + exposureDelta); });
```

Comment:
I think it was an experiment to see if it would improve performance. Looking at it now it doesn't seem to have any clear impact, so I've removed it.  thanks for catching that.